index.html

```diff
@@ -5298,7 +5297,7 @@ <h3>Adding Protocol Bindings</h3>
 {
     "@context": [
         "https://www.w3.org/2022/wot/td/v1.1",
-        { "cov": "http://www.example.org/coap-binding#" }
+        { "cov": "https://www.w3.org/TR/wot-coap-binding/" }
```

Comment:
alternative namespace: https://www.w3.org/TR/wot/binding/coap10

@ashimura would be this possible?